app/src/main/java/com/infomaniak/mail/ui/newMessage/AiPromptFragment.kt

```diff
@@ -126,12 +126,20 @@ class AiPromptFragment : Fragment() {
                 setText(aiViewModel.aiPrompt)
                 setSelection(length())
 
-                val promptExample = getString(promptExamples.random())
-                hint = getString(R.string.aiPromptPlaceholder, promptExample)
+                val isAnswering = newMessageViewModel.previousMessageBodyPlainText != null
```

Comment:
I saw another PR where all `answering` is replaced by `replying`.
Do we want to do it here too?